middleware/wrap_writer.go

```diff
@@ -98,6 +103,8 @@ func (b *basicWriter) Write(buf []byte) (n int, err error) {
 		}
 	} else if b.tee != nil {
 		n, err = b.tee.Write(buf)
+	} else {
+		n, err = io.Discard.Write(buf)
```

Comment:
https://github.com/go-chi/chi/actions/runs/9712399165/job/26807549159?pr=926

Looks like we'll need to use `ioutil.Discard` because of backward-compatibility with older versions of Go